lib/Subscriber/Behavior/Audit/TimestampSubscriber.php

```diff
@@ -45,18 +46,23 @@ public function handleMetadataLoad($event)
             return;
         }
 
+        $encoding = 'system_localized';
+        if ($event->getMetadata()->getReflectionClass()->isSubclassOf(NonLocalizedTimestampBehavior::class)) {
+            $encoding = 'system';
+        }
```

Comment:
This would be the first subscriber not being directly related to a single Behavior (this was quite nice until now IMO, because it was always clear where to look for the operations executed because of these behaviors).

@dantleech Do you have any better ideas? For me it would be even better to use inheritance here (an `AbstractTimestampBehavior` which is the base class for a `TimestampBehavior` and a `LocalizedTimestampBehavior`, whereby the derived classes only differ in their usage of the encoding property).